js/form-validation.js

```diff
@@ -0,0 +1,23 @@
+// Cliente side validation
+
+// Get elements for the form, need the actual form, the email
+// and the div id to show the error.
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const errorElement = document.getElementById('error');
+
+form.addEventListener('submit', (event) => {
+  const messages = [];
+  if (email.value.toLowerCase() !== email.value) {
+    messages.push('Please enter your email address in lowercase letters.');
+  }
+
+  if (messages.length > 0) {
+    event.preventDefault();
+    errorElement.innerText = messages.join(', ');
+    // Remove the message after 3 seconds
+    setTimeout(() => {
+      errorElement.remove();
```

Comment:
[OPTIONAL] Well done on the toast 💭 but removing the whole element here makes the error stop appearing even if the email still contains uppercase characters. Editing the `innerText` like below fixes this:
```
errorElement.innerText = '';
```